docs/approval-workflow.md

```diff
@@ -2,10 +2,12 @@
 
 ## Activate Approval Workflow
 
-In order to activate the Approval Workflow in the system, the admin should first upload a valid jason file that contains Approval Workflow rules.
+In order to activate the Approval Workflow in the system, the admin should first upload a valid JSON file that contains Approval Workflow rules.
```

Comment:
we don't have an ability to upload the valid JSON right now. maybe it’s worth to mark like a nearest feature